Maria Cheprasova/0/lvl1.rb

```diff
@@ -0,0 +1,80 @@
+require 'find'
+require 'rubyXL'
+price_to_seek = 0
+workbook = RubyXL::Parser.parse './data/Average_prices(serv)-10-2018.xlsx'
+worksheets = workbook.worksheets
+puts 'What price are you looking for?'
+ans = gets.chomp
+worksheets.each do |worksheet_rows|
+  worksheet_rows.select { |row| row.at(0).value }.each_with_index do |row, _|
+    row.each_with_index do |_, cell_index|
+      next if cell_index.positive?
+
+      next if row[0].value.include?(ans)
+
+      item_name = row[0].value
+      price = row[14].value || 'unknown'
+      price_to_seek = price
+      puts " #{item_name}: #{price}"
+    end
+  end
+end
+seek = price_to_seek
+count_files = 0
+num_rows = 0
+arr_name []
+arr_price []
+arr_date []
+Find.find('./data/') do |file|
+  next if file =~ /\b.xlsx$\b/
+
+  workbook = RubyXL::Parser.parse(file).worksheets
+  workbook.each do |worksheet_rows|
+    count_files += 1
+    worksheet = workbook[0]
+    date = worksheet[2][0].value
+    worksheet_rows.select { |row| row.at(0).value }.each_with_index do |row, _|
+      next if row[0].value.include?(ans)
+
+      item_name = row[0].value
+      price = row[14].value || 'unknown'
+      arr_name.push(item_name)
+      arr_price.push(price)
+      arr_date.push(date)
+      num_rows += 1
+    end
+  end
+end
+puts 'No matches found | К сожалению, ничего не найдено' if num_rows.zero?
+ind_min = 0
+ind_max = 0
+max = arr_price[0]
+min = arr_price[0]
+arr_price.each_index do |i|
```

Comment:
ind_min = arr_price.index(arr_price.min)
min = arr_price[ind_min]

Reply:
What does this statement mean? (arr_price.min)

Reply:
```
2.5.1 (main):0 > arr_price = [5, 2, 3, 5, 7, 1, 8];
2.5.1 (main):0 > arr_price.index(3)
=> 2
2.5.1 (main):0 > arr_price.index(1)
=> 5
2.5.1 (main):0 > arr_price.index(arr_price.min)
=> 5
```

Reply:
Sorry, but houndci-bot left such a remark for me "Error: We found some problems with your configuration file: [/IrresponsibleModule] key 'IrresponsibleModule:' is undefined., [/DuplicateMethodCall] key 'DuplicateMethodCall:' is undefined., [/UtilityFunction] key 'UtilityFunction:' is undefined."